jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationDialog.kt

```diff
@@ -339,7 +327,7 @@ class DeployServerlessApplicationDialog(
             }
         }
 
-    private fun refreshTemplateParameters(updateStackName: String? = null) {
+    private fun refreshTemplateParametersAndTags(updateStackName: String? = null) {
```

Comment:
Can we drop "update" from parameter name, it sounds like it will mutate stack name